elm_deps_sync.py

```diff
@@ -2,55 +2,39 @@
 from __future__ import print_function
 
 import sys
-from collections import OrderedDict
-import json
 import argparse
 
+import elm_package
```

Comment:
Yea I guessed so :p I didn't see myself downloading a single file without the option to easily upgrade: I'd do either pip or a git clone